### DISCUSSION TITLE
Time lapse sync

### DISCUSSION THREAD

**Discussion by Sirnut** [nlef/moonraker-telegram-bot#101] [Ideas]
Is it possible to have Timelapse photos taken when axis are in the same position? Similar to octolapse

**Comment by aka13-404** [comment]
Yes, refer to [manual timelapse modes](https://github.com/nlef/moonraker-telegram-bot/wiki/Interacting-with-klipper#controlling-timelapse-parameters-via-gcode). 
You can disable the automated timelapse pictures, and instruct the bot to take pictures whenever (or wherever) you want them to be taken.

**Reply by Sirnut** [reply]
I've spent some time looking into the settings but I'm not quite sure that I understand

Ideally I would like to be able to tell the bot to take an image for the timelapse when the printer is at a specific X/Y

My camera is mounted to the X gantry on my ender 3V2 so the ability to sync the photos being taken would be very convenient

**Reply by aka13-404** [reply]
Well, yo have to
1. Disable the automated tamelaps e picture taking (since you obviously want the pictures be at a specific point in time under specific conditions)
2. Drive/Park the head wherever you want with a macro caled from slicer on every layer
3. Instruct to take the bot to take the picture when you are in the desired position and then proceed with printing.
